src/core/probe/kernel/inspect.rs

```diff
@@ -30,6 +30,22 @@ pub(super) fn inspect_symbol(symbol: &Symbol) -> Result<ProbeConfig> {
     Ok(cfg)
 }
 
+/// Find the offset of drop reason enums. We do not distinguish them as they're
+/// part of the same (virtual) enum.
+fn drop_reason_offset(symbol: &Symbol) -> Result<Option<i8>> {
+    Ok(
+        if let Some(offset) = symbol.parameter_offset("enum skb_drop_reason")? {
+            Some(offset as i8)
+        } else if let Some(offset) = symbol.parameter_offset("enum mac80211_drop_reason")? {
+            Some(offset as i8)
+        } else {
```

Comment:
why not an explicit check here as well?

Reply:
sure. Had no doubt about the behavior, the point was the readability as you said.
Let's just make Clippy happy without adding exceptions.

Reply:
ok :)
I was assuming you didn't want to add the exception. Had no strong feelings about it, but I also slightly prefer to allow this case. Thanks